packages/components/socialFeed/NewsFeed/NewsFeedQueries.ts

```diff
@@ -161,17 +161,58 @@ export const createPost = async ({
     mentions: formValues.mentions || [],
   });
 
-  await client.createPost(
-    {
+  const network = mustGetNetwork(networkId);
+
+  if (network.kind === NetworkKind.Gno) {
+    const msg = {
       category,
-      identifier: identifier || uuidv4(),
+      identifier,
       metadata: JSON.stringify(metadata),
       parentPostIdentifier: parentId,
-    },
-    defaultSocialFeedFee,
-    "",
-    freePostCount ? undefined : [coin(fee, "utori")]
-  );
+    };
+
+    const vmCall = {
+      caller: wallet.address,
+      send: "",
+      pkg_path: GNO_SOCIAL_FEEDS_PKG_PATH,
+      func: "CreatePost",
+      args: [
+        TERITORI_FEED_ID,
+        msg.parentPostIdentifier || "0",
+        msg.category.toString(),
+        msg.metadata,
+      ],
+    };
+
+    const txHash = await adenaDoContract(
+      networkId,
+      [{ type: "/vm.m_call", value: vmCall }],
+      {
+        gasWanted: 2_000_000,
```

Comment:
?

Reply:
yes, 1M does not work so I hardcode to one value that works to make user flow easier, for now I don't see another way to estimate gas efficiently

Reply:
we can't simulate tx with gno yet